android/java/org/chromium/chrome/browser/app/BraveActivity.java

```diff
@@ -2389,27 +2396,29 @@ public void addOrEditBookmark(final Tab tabToBookmark) {
     // We call that method with an interval
     // BraveSafeBrowsingApiHandler.SAFE_BROWSING_INIT_INTERVAL_MS,
     // as upstream does, to keep the GmsCore process alive.
-    private void executeInitSafeBrowsing(long delay) {
-        // SafeBrowsingBridge.getSafeBrowsingState() has to be executed on a main thread
-        PostTask.postDelayedTask(
-                TaskTraits.UI_DEFAULT,
-                () -> {
-                    SafeBrowsingBridge safeBrowsingBridge =
-                            new SafeBrowsingBridge(getCurrentProfile());
-                    if (safeBrowsingBridge.getSafeBrowsingState()
-                            != SafeBrowsingState.NO_SAFE_BROWSING) {
-                        // initSafeBrowsing could be executed on a background thread
-                        PostTask.postTask(
-                                TaskTraits.USER_VISIBLE_MAY_BLOCK,
-                                () -> {
-                                    BraveSafeBrowsingApiHandler.getInstance().initSafeBrowsing();
-                                });
-                    }
-                    executeInitSafeBrowsing(
-                            BraveSafeBrowsingApiHandler.SAFE_BROWSING_INIT_INTERVAL_MS);
-                },
-                delay);
-    }
+
+    // TODO(alexeybarabash): needs to be redone for cr130
```

Comment:
@AlexeyBarabash the plan is to merge as is and fix in a follow-up?

Reply:
@iefremov 
yes, the plan is to merge as is and  create a follow-up PR